### ISSUE TITLE
"Use multivalue separator" for all fields

### ISSUE THREAD

**Issue by andrisi** [Daniel-KM/Omeka-S-module-BulkImport#7]
I'm importing a CSV with a vocabulary with almost a 100 fields. Auto mapping works great, but it seems that to import multivalue fields I have to check the "Use multivalue separator" setting for each field on every (re)import which is tedious. Could we have a global setting under the "Advanced settings" tab that would make all imported fields multivalue aware? Or have a special field name pattern, like "vocabulary:fieldname:multi" for multivalue fields? Thanks!

**Comment by Daniel-KM** [comment]
Hi, 
This is the default feature : set the separator in first form and it will be checked in all columns.

**Comment by andrisi** [comment]
I did set a separator, and it did not separate multiple values. Also, what is the point of the per field setting, if the default is to check in all? Perhaps if on the next page all "use multivalue separator" checkboxes would be checked by default if I separator? That would be helpful. But for now, only in fields for which I checked it manually was it honored. And it was, so the separator I set matched the one in the data.
